test/artifact/inspec_supermarket_test.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "artifact_helper"
+
+class TestInspecSupermarket < Minitest::Test
+  parallelize_me!
+
+  def test_supermarket
+    flunk
```

Comment:
should probably be a skip... this is on a fresh pipeline, so even an `S` output will stand out and you'll want to fix them.